src/plone/rest/tests/test_explicitacquisition.py

```diff
@@ -11,18 +11,17 @@
 
 import unittest
 
-
 try:
-    from Products.CMFCore.interfaces import IShouldAllowAcquiredItemPublication
+    from Products.CMFCore.explicitacquisition import SKIP_PTA
```

Comment:
@jaroel I am wondering if we should use the abbreviation "SKIP_PTA" here. By just looking at the PR I have no clue what this is about. I'd need to look into CMFCore to understand what it means and does. Though, I am not sure if this is still possible to change without breaking too many things...

Reply:
@tisto Don't think it'll be a problem. It's probably not used anywhere yet. CMFCore >= 3.2 is very recent and we made the PR.